.github/workflows/deployment.yml

```diff
@@ -72,18 +72,24 @@ jobs:
       environment: ${{ env.ENVIRONMENT }}
       timezone: ${{ env.TIMEZONE }}
     steps:
+      - name: Configure 🔨
+        if:  ${{ !contains(fromJson('["feature", "staging", "prod"]'), github.event.workflow_run.head_branch) }}
+        run: |
+          echo "ENVIRONMENT=dev" >> $GITHUB_ENV
```

Comment:
So this just overrides it from head_branch to dev if it's not feature/staging/prod? is this just a bit of a plaster fix for getting release-2.0 to go to dev but no where else?

Reply:
Correct, this is for all future `release-*` to use `dev` environment.